packages/flutter_tools/lib/src/desktop_device.dart

```diff
@@ -270,7 +270,7 @@ abstract class DesktopDevice extends Device {
     // tool and the device, usually in debug or profile mode.
     if (debuggingOptions.debuggingEnabled) {
       if (debuggingOptions.deviceVmServicePort != null) {
-        addFlag('observatory-port=${debuggingOptions.deviceVmServicePort}');
+        addFlag('vmService-port=${debuggingOptions.deviceVmServicePort}');
```

Comment:
I believe this should be `vm-service-port`

https://github.com/flutter/engine/blob/cd2e8885e491a8e082ee98a00686a1f983b1f142/shell/common/switches.h#L87

Reply:
Good catch, thanks! There was a lot of `s/observatory/vmService/` happening, I must have missed this one.

Reply:
Resolved. PTAL.